File: backend/Models/Track.cs
namespace backend.Models
{
    public class Track
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int AlbumId { get; set; }
        public string Composer { get; set; }
        public int Milliseconds { get; set; }
        public DateTime ReleaseDate { get; set; }
        public Album Album { get; set; }
        public string AddedBy { get; set; }
    }
}

File: frontend/src/models/Track.ts
import Album from "./Album";

export default interface Track {
    id?: number;
    name: string;
    albumId: string;
    composer: string;
    milliseconds: number;
    releaseDate: Date;
    appearsInPlaylists: number;
    addedBy: string;
    album : Album;
}